dips/dip-158.md

```diff
@@ -1,42 +1,36 @@
 ---
 dip: 158
-title: P2M (eCommerce) Payments Off-Chain Protocol
+title: P2M Charge Payment - Off-Chain Protocol
 authors: Yinon Zohar, Daniel Prinz, David Wolinsky, Dimitri Roche
 status: Draft
 type: Standard
 created: 03/15/2021
-updated: 05/14/2021
+updated: 05/23/2021
 requires: DIP-1, DIP-5, DIP-161
 ---
 
 # Summary
-This DIP leverages the off-chain protocol defined in [DIP-1](https://github.com/diem/dip/blob/main/dips/dip-1.mdx) and extends it to enable eCommerce P2M payments (Peer To Merchant) of the following types:
-* Charge
-* Auth / Capture
+This DIP leverages the off-chain protocol defined in [DIP-1](https://github.com/diem/dip/blob/main/dips/dip-1.mdx) and extends it to enable P2M Charge payment.
 
-This DIP defines the type of eCommerce P2M payments, corresponding requests and the payment structures. 
+This DIP defines the type of the P2M payments, corresponding requests and the payment structures. 
  
-This DIP makes the Auth/Capture section in [DIP-8](https://dip.diem.com/dip-8/#authcapture) redundant. The auth/capture section in DIP-8 is to be disregarded and will be removed in a future version of DIP-8. 
-
 This DIP does NOT include payment flows which require funds pull pre-approval such as subscription and recurring payments. 
 
 This DIP makes use of the `result` field added to the `CommandResponseObject` as described in [DIP-161](https://github.com/diem/dip/blob/main/dips/dip-161.md)
 
 ---
 # Abstract / Motivation
-This DIP is an extension of the Off-Chain Protocol to support more advanced payment functionality - particularly eCommerce P2M payments. 
+This DIP is an extension of the Off-Chain Protocol to support more advanced payment functionality - particularly eCommerce P2M Charge payment.
+Charge payment stipulates an immediate capture of funds. This means that a successful off-chain negotiation for such payment will result in funds being captured (and submit a corresponding transaction on-chain and actually transfer the funds). 
+
 ## How P2M Payments Differ from P2P Transfers 
 There are several key differences between the process of paying a merchant (P2M) and a simple P2P transfer:
 ||P2M|P2P|
 |-|-|-|
 |**Payer Data**|Customer details are required for **every** payment / purchase in order to perform risk checks and apply AML policies|Required only for transfers for which Travel Rule applies. i.e., over 1000 USD|
```

Comment:
For Charge payments, there are reasons to remove the transfer of Payer Data
1. Instant settlement reduces the risk profile such that only the customer is at risk. The instant transfer is great for the merchant VASP.
1. This is different to auth and cap, which we are not addressing in this DIP.
1. The only issue here may be with the SDN list, but that should be addressed by the merchant site.

I want to suggest that the introduction and transfer of Payer Data comes in the DIP associated with auth an cap. Let's discuss offline.

Reply:
The timing of the settlement (immediate or delayed) is irrelevant.
Payer data is required in both scenarios and should be part of DIP-158 (as decided last week)